addon/components/o-s-s/upload-area.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if this.hasFile}}
+{{else}}
```

Comment:
```suggestion
{{#unless this.hasFile}}
```

Reply:
@JulienVannier66 That's where i plan to display the uploaded file, that's why i let the `if` even if empty